dbdreader/dbdreader.py

```diff
@@ -485,20 +485,20 @@ def __init__(self):
                        'the8x3_filename':'string'}
         self.info={}
 
-    def read_header(self,fp):
+    def read_header(self, fp, filename=''):
```

Comment:
not sure how fussy you are about API changes - this is theoretically one, though I suspect "read_header" is most an internal utility versus something the end user would use...